hypothesis-python/src/hypothesis/internal/conjecture/providers.py

```diff
@@ -331,10 +383,27 @@ def draw_integer(
         min_value: Optional[int] = None,
         max_value: Optional[int] = None,
         *,
-        # weights are for choosing an element index from a bounded range
         weights: Optional[dict[int, float]] = None,
         shrink_towards: int = 0,
     ) -> int:
+        """
+        Draw an integer choice.
+
+        Parameters
+        ----------
+        min_value : int | None
+            (Inclusive) lower bound on the integer value. If ``None``, there is
+            no lower bound.
```

Comment:
We're inconsistent about (Inclusive) vs (inclusive); let's pick one and stick to it.